src/bot.js

```diff
@@ -86,12 +86,11 @@ let initializeBot = () => {
 			if (command === 'tft' && data) {
 				logger.info('tft command...');
 				message.channel.send(('https://tracker.gg/tft/profile/riot/NA/'+data+'/overview'));
-
 			}
 		}
 	});
 
-	client.login(auth.token);
+	client.login(config.token);
 };
 
 module.exports = {initializeBot};
```

Comment:
```suggestion
module.exports = {initializeBot};

```

Reply:
Is there supposed to be the space? I think the linter removed the space 🤔

Reply:
second comment [here](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file/5813359) is good

I complained cause Github complains 😆

Reply:
I dunno how to add it correctly without making an extra line, my vim just does it when I save...
Did you change your editor end of line settings or something?